src/migrate/Migrator.js

```diff
@@ -154,7 +157,8 @@ class Migrator {
         })
         .then((migrations) => {
           return this._runBatch(migrations, 'down');
-        });
+        })
+        .then(resolve, reject);
```

Comment:
we don't need catch block for this chain to reject the promise?

Reply:
no the `then(resolve, reject` will catch the failed promise and pass the arguments to `reject`, which will propogate the failure to the caller `catch`